src/main/kotlin/org/kamiblue/client/module/modules/chat/FancyChat.kt

```diff
@@ -14,14 +14,14 @@ internal object FancyChat : Module(
     showOnArray = false,
     modulePriority = 100
 ) {
-    private val uwu = setting("uwu", true)
-    private val leet = setting("1337", false)
-    private val mock = setting("mOcK", false)
-    private val green = setting(">", false)
-    private val blue = setting("`", false)
-    private val randomSetting = setting("Random Case", true, { mock.value })
-    private val commands = setting("Commands", false)
-    private val spammer = setting("Spammer", false)
+    private val uwu = setting("uwu", false, description = "Uwufy all of your messages")
+    private val leet = setting("1337", false, description = "Exchange certain characters for other similar looking characters")
+    private val mock = setting("mOcK", false, description = "Change from lower case to upper case")
```

Comment:
This description would imply it changes all the letters to upper case, which, in reality, it changes letters to a random case